packages/codefixes/src/codefixes-provider.ts

```diff
@@ -12,17 +12,14 @@ export class CodeFixesProvider {
   }
 
   /**
-   * Returns a code action contains text changes to fix the diagnosed issue
+   * Returns all available to user code actions containing text changes to fix the diagnosed issue
    */
-  getCodeFixes(diagnostic: DiagnosticWithContext): CodeFixAction | undefined {
+  getCodeFixes(diagnostic: DiagnosticWithContext, onlySafeFixes: boolean): CodeFixAction[] {
+    let fixActions: CodeFixAction[] = [];
     for (const collection of this.collections) {
-      const action = collection.getFixForDiagnostic(diagnostic);
-      // Return the first available CodeFixAction
-      if (action !== undefined) {
-        return action;
-      }
+      fixActions = [...fixActions, ...collection.getFixesForDiagnostic(diagnostic, onlySafeFixes)];
```

Comment:
Merge all available codefixes from all providers to display to user in the interactive mode